dev-workflow-ui-web-test/src_test/ch/ivyteam/ivy/project/workflow/webtest/util/WorkflowUiUtil.java

```diff
@@ -55,31 +55,30 @@ private static String viewUrl(String page) {
   }
 
   public static void assertCurrentUrlContains(String contains) {
-    WebDriverConditions.currentFrameUrlContaining(contains);
```

Comment:
This never does any assertion, this is only a condition.
Fixing this has broken over half of all tests...